tools/data-handler/test/command-handler-create.test.ts

```diff
@@ -392,6 +392,23 @@ describe('create command', () => {
     );
     expect(result.statusCode).to.equal(400);
   });
+  it('create a label (success)', async () => {
+    const result = await commandHandler.command(
+      Cmd.create,
+      ['label', 'decision_6', 'test'],
+      options,
+    );
+    expect(result.statusCode).to.equal(200);
+  });
+
+  it('try create a label - label exists', async () => {
```

Comment:
Add tests for invalid names (empty strings + if you add any other limitations; length, invalid characters).

Reply:
added those tests